packages/blade/src/components/Input/PhoneNumberInput/PhoneNumberInput.stories.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import type { StoryFn, Meta } from '@storybook/react';
+import { isValidPhoneNumber } from '@razorpay/i18nify-js';
+import type { PhoneNumberInputProps } from './types';
+import { PhoneNumberInput } from './PhoneNumberInput';
+import { Box } from '~components/Box';
+import { Code, Text } from '~components/Typography';
+import { PhoneIcon } from '~components/Icons';
+
+const meta: Meta<PhoneNumberInputProps> = {
+  title: 'Components/Input/PhoneNumberInput',
+  component: PhoneNumberInput,
+  tags: ['autodocs'],
+};
+
+const PhoneNumberInputTemplate: StoryFn<typeof PhoneNumberInput> = ({ ...args }) => {
+  return <PhoneNumberInput {...args} />;
+};
+
+export const Default = PhoneNumberInputTemplate.bind({});
+
+const CountriesToShowTemplate: StoryFn<typeof PhoneNumberInput> = ({ ...args }) => {
+  return (
+    <Box>
+      <Text marginBottom="spacing.5">
+        By setting the <Code size="medium">{`countries={['IN', 'MY']}`}</Code> prop, We can only
+        show two countries in the Country Selector
+      </Text>
+      <PhoneNumberInput {...args} />
+    </Box>
+  );
+};
+export const CountriesToShow = CountriesToShowTemplate.bind({});
+CountriesToShow.args = {
+  countries: ['IN', 'MY'],
+};
+
+export const SizeLarge = PhoneNumberInputTemplate.bind({});
+SizeLarge.storyName = 'Size: Large';
+SizeLarge.args = {
+  size: 'large',
+};
+
+export const WithoutCountrySelector = PhoneNumberInputTemplate.bind({});
+WithoutCountrySelector.args = {
+  showCountrySelector: false,
+};
+
+export const WithoutDialCode = PhoneNumberInputTemplate.bind({});
+WithoutDialCode.args = {
+  showDialCode: false,
```

Comment:
does this have valid usecase? looks a bit strange in UI 🤔

Reply:
design mein people can remove the prefix text. Not sure about usecase.